doc/source/operations/bifrost-hardware-inventory-management.rst

```diff
@@ -0,0 +1,277 @@
+=====================================
+Bifrost Hardware Inventory Management
+=====================================
+
+In most deployments, hardware inventory is managed by the Bifrost service.
+
+Reconfiguring Control Plane Hardware
+====================================
+
+If a server's hardware or firmware configuration is changed, it should be
+re-inspected in Bifrost before it is redeployed into service. A single server
+can be reinspected like this:
+
+.. code-block:: console
+
+   kayobe# kayobe overcloud hardware inspect --limit <Host name>
+
+.. _enrolling-new-hypervisors:
+
+Enrolling New Hypervisors
+-------------------------
+
+New hypervisors can be added to the Bifrost inventory by using its discovery
+capabilities. Assuming that new hypervisors have IPMI enabled and are
+configured to network boot on the provisioning network, the following commands
+will instruct them to PXE boot. The nodes will boot on the Ironic Python Agent
+kernel and ramdisk, which is configured to extract hardware information and
+send it to Bifrost. Note that IPMI credentials can be found in the encrypted
+file located at ``${KAYOBE_CONFIG_PATH}/secrets.yml``.
```

Comment:
Could you remove the brackets from the references to KAYOBE_CONFIG_PATH (and other env vars) throughout?
```suggestion
file located at ``$KAYOBE_CONFIG_PATH/secrets.yml``.
```